src/python/include/Persistent_cohomology_interface.h

```diff
@@ -73,6 +72,54 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  int top_dimensional_coface(int splx){
+    if (stptr_->dimension(splx) == stptr_->dimension()){return splx;}
+    else{  
+      for (auto v : stptr_->coboundary_simplex_range(splx)){
+        if(stptr_->filtration(v) == stptr_->filtration(splx)){
+          return top_dimensional_coface(v);
+        }
+      }  
+    }
+  }
+
+  std::vector<std::vector<int>> cofaces_of_cubical_persistence_pairs() {
```

Comment:
Since the inner vectors all have the same size, why not `std::vector<std::array<XXX>>` or `std::vector<std::pair<XXX>>`?

Reply:
I am not sure how to use C++ array in Cython. Is it the same than vectors and pairs?

Reply:
Indeed, cython cannot handle array/tuple... A `vector<pair<int,pair<size_t,size_t>>>` might work, but let's postpone that.